### PR TITLE
Add KDE Connect macOS receipt

### PR DESCRIPTION
**Pull request by Inokinoki**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

As to Home Page, currently, my GitHub repo is for macOS version release. To match the `url`, `appcast` and `hompage`, I use my GitHub repo as the home page.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

For the name, `kdeconnect` is the official name. So, the name is adopted to make the receipt.

### CODE REVIEW

File: Casks/kdeconnect.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'kdeconnect' do
+  version '20190901'
+  sha256 '871d0be35e931b1ce5eee37eb4ca6927557484bf158c1ab46727f0e8939a34f9'
+
+  url 'https://github.com/Inokinoki/kde-blog/releases/download/20190901/kdeconnect-indicator-no-qca.zip'
```
Comment (miccal) [inline Casks/kdeconnect.rb:5]: Please change this to:
```
url "https://github.com/Inokinoki/kde-blog/releases/download/#{version}/kdeconnect-indicator-no-qca.zip"
```

File: Casks/kdeconnect.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  appcast 'https://github.com/Inokinoki/kde-blog/releases'
```
Comment (miccal) [inline Casks/kdeconnect.rb:6]: Please change this to:
```
appcast 'https://github.com/Inokinoki/kde-blog/releases.atom'
```

Comment (Inokinoki) [reply Casks/kdeconnect.rb]: Thanks for your comments!
I'm getting to know better Ruby and HomeBrew receipt

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

**Comment by Inokinoki** [comment]
> Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):
> 
> > * The Cask is too obscure. Examples:
> >   
> >   * A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
> >   * [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

I'm sorry but the [origin project](https://github.com/KDE/kdeconnect-kde) has 800+ stars on GitHub.

**Pull request closed without merging**